style.css

```diff
@@ -327,10 +368,90 @@ footer p {
   height: auto;
 }
 
+.bar {
+  display: block;
+  width: 80%;
+  margin-inline: 10%;
+}
+
+.details {
```

Comment:
- [ ] [OPTIONAL] The position of fixed here doesn't make your modal scrollable, therefore not enabling the user to see the rest of your modal that include the buttons.
<img width="960" alt="2022-07-23 (3)" src="https://user-images.githubusercontent.com/52934128/180587592-f66bdc30-3231-46f8-a0f9-04b7ade9dcb6.png">